src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -687,6 +687,8 @@ public static void hoverInAppServerCfgFile(RemoteRobot remoteRobot, String hover
             } catch (WaitForConditionTimeoutException wftoe) {
                 error = wftoe;
                 TestUtils.sleepAndIgnoreException(2);
+                // click on center of editor pane - allow hover to work on next attempt
```

Comment:
The change looks ok, but going back a few lines, line 683, i think the if statement needs a continue if the intent there is to try again. Otherwise, you need to do something with the break on 686. It also seems that the if in question is missing and else or a some code to check for a specific text?

Reply:
This method is designed to just raise the hover popup window itself. The main testcase goes on to call additional methods to get the popup text and then validate the popup text. I agree some consolidation would be appropriate, I can do that in a future cleanup issue.

Reply:
Agreed, we should either be doing some sort of confirmation in `hoverInAppServerCfgFile()` or we should consolidate with `getHoverStringData()`. Let's try to address that in a future PR.